src/_components/Practice.jsx

```diff
@@ -1,10 +1,59 @@
+function getIconForPractice(number, content) {
+  const lowerContent = content.toLowerCase();
+
+  if (number === 1 && lowerContent.includes("insurance")) {
```

Comment:
Instead of these if statements, in practices.md you can edit the metadata:

---
bestPractices:
  - content: If you are insured, understand the terms of your insurance plan with respect to deductibles, co-pays, co-shares and whether you need prior approval.
     icon: "/assets/vector/name.svg"
 ---
 
 and than you can pass that through the BestPractices.jsx layout into this component as a prop. This way you can get rid of this whole if statement block and it'll make it easier to edit icons.

Reply:
A good example is the FAQ page ^